src/js/rivet-collapsible.js

```diff
@@ -151,20 +151,28 @@
       switch (event.keyCode) {
         // Up arrow key
         case keys.up:
-          // If the current active toggle is the first bail
-          if (accordionTogglesArr[prevToggle] === undefined) return;
-
-          // Focus the previous toggle
-          accordionTogglesArr[prevToggle].focus();
+          if (!event.target.hasAttribute(DATA_ATTR)) {
+            return
```

Comment:
Small thing here, but we're using semicolons consistently so we should probably have one here too.